audit-connector/src/test/java/com/sequenceiq/cloudbreak/audit/converter/AuditEventToGrpcAuditEventConverterTest.java

```diff
@@ -115,13 +139,20 @@ void convertWithApiRequestData() {
                 .build();
         AuditEvent source = makeAuditEvent(actor, eventData);
 
+        underTest = new AuditEventToGrpcAuditEventConverter(createMockUtilizer(ApiRequestData.class));
+
         AuditProto.AuditEvent target = underTest.convert(source);
         assertGeneric(target);
-        assertThat(target.getEventTypeCase()).isEqualTo(AuditProto.AuditEvent.EventTypeCase.APIREQUESTDATA);
-        assertThat(target.getApiRequestData().getApiVersion()).isEqualTo(API_VERSION);
-        assertThat(target.getApiRequestData().getMutating()).isEqualTo(MUTATING);
-        assertThat(target.getApiRequestData().getRequestParameters()).isEqualTo(REQUEST_PARAMETERS);
-        assertThat(target.getApiRequestData().getUserAgent()).isEqualTo(USER_AGENT);
```

Comment:
I miss these tests in for the recently refactored converter classes

Reply:
you're right, I'll make them reappear in new test cases. same goes for the other requests as well